lib/bots/versionbot.ts

```diff
@@ -239,21 +239,23 @@ const StatusAutoMerge = {
 export class VersionBot extends ProcBot {
 	/** Github ServiceListener name. */
 	private githubListenerName: string;
-	/** Github ServiceEmitter name. */
+	/** Github ServiceListener name. */
 	private githubEmitterName: string;
```

Comment:
What's going on here? You have the comment saying that it's a `Listener`, but the varname is still `Emitter`.